src/conf.rs

```diff
@@ -645,6 +658,58 @@ impl<MODE> MpuConfig<MODE> {
         self.sample_rate_divisor = Some(smplrt_div);
         self
     }
+
+    #[cfg(not(feature = "i2c"))]
+    /// Build new Mpu9250 from current configuration using the provided device
+    pub fn build<E, SPI, NCS>(self,
```

Comment:
For SPI connected devices we also provide `*_with_reinit` family of constructors. Although they are a bit ugly, they serve practical need -- maybe worth adding support for reinit to builders?

Reply:
Could you give me an example of how you would use `*with_reinit()` so I can try to design something ergonomic.